### ISSUE TITLE
Modifying touch listener.

### ISSUE THREAD

**Issue by pawarprasad37** [chrisbanes/PhotoView#573]
I've placed the PhotoView inside a card swipe container. If you swipe the card left or right similar to Tinder app, the card is dismissed. What I need is: 
- If the image is zoomed out(scale = 1.0) and user swipes the card container(with one finger), the touch event should go to the card container. 
- Pinch zoom should zoom the child PhotoView. 
- If scale > 1.0 the PhotoView should scroll horizontally or vertically.

So in conclusion, for one finger gesture and scale=1.0, i want the card container to consume touch event.
By default, the PhotoView is consuming all touch events. How can I modify the library to make this happen?
